src/data/faq.json

```diff
@@ -116,6 +116,62 @@
+                        ]
+                    },
+                    {
+                        "title": "How can I request a test certificate?",
+                        "anchor": "test_cert_get",
+                        "active": true,
+                        "textblock": [
+                            "You can request the certificate by registering a test in the Corona-Warn-App (CWA): To do this, you can tap on 'Register Test' on the star screen of your CWA and then on 'Scan QR Code'. After scanning the QR code for test registration, the window for the COVID test certificate automatically opens. There you  can 'request the test certificate' or reject it if you do not want to receive one."
+                        ]
+                    },
+                    {
+                        "title": "Can I request test certificates for rapid tests and PCR tests?",
+                        "anchor": "test_cert_rat_and_pcr",
+                        "active": true,
+                        "textblock": [
+                            "You can request a certificate for both PCR and rapid tests, which is then issued in the event of a negative test.",
+                            "<b>Rapid tests</b><br>In the case of rapid tests, it may be that the respective test site does not support the digital certificate. In this case, after scanning the QR code, you will not be redirected to apply for the certificate. You will then receive the information that a test certificate cannot be requested because the test site does not support the issuance of test certificates.","<b>PCR tests</b><br>When requesting a test certificate for a PCR test, you must provide your date of birth. It is important that the date of birth corresponds to the one that was also indicated for the sample of PCR test. If the dates of birth do not match, for example due to a typo, you will not receive the test result or the certificate via the app. You cannot change the date of birth that you enter in the app afterwards. Before you request the test certificate after entering the date, you should therefore briefly check whether the date is correct."
+                        ]
+                    },
+                    {
+                        "title": "When will I receive my test certificate and where can I find it?",
+                        "anchor": "test_cert_when_and_where",
+                        "active": true,
+                        "textblock": [
+                            "Shortly after the test result is available in the app, you will also receive your test certificate in the app - provided that you have requested it beforehand, and the respective rapid test site supports the creation of a certificate. You can then view it via the 'Certificates' tab in your tab. If they tap on the COVID test certificate, they will be shown the QR code that can be scanned for verification, and more information, such as type of test, date and time of execution and test result."

```

Comment:
"'Certificates' tab in your tab" ?
Not sure what "your tab" means.

"If they tap" In the previous sentence it was "you", so who is now "they"?